src/app/templates/client/Client.jsx

```diff
@@ -16,11 +16,33 @@ import navigation from '../../../client/state/navigation';
 import cons from '../../../client/state/cons';
 import DragDrop from '../../organisms/drag-drop/DragDrop';
 
+const viewPossibilities = {
+  nav: 'navigation',
+  room: 'room',
+};
+
 function Client() {
   const [isLoading, changeLoading] = useState(true);
   const [loadingMsg, setLoadingMsg] = useState('Heating up');
   const [dragCounter, setDragCounter] = useState(0);
+  const [activeView, setActiveView] = useState(viewPossibilities.nav);
+
+  // #region Liston on events for compact screen sizes
+  const onRoomSelected = () => setActiveView(viewPossibilities.room);
+  const onNavigationSelected = () => setActiveView(viewPossibilities.nav);
+
+  useEffect(() => {
+    navigation.on(cons.events.navigation.ROOM_SELECTED, onRoomSelected);
+    navigation.on(cons.events.navigation.OPEN_NAVIGATION, onNavigationSelected);
+    // appDispatcher.register(cons.events.navigation.ROOM_SELECTED, () => setOnNavRoom(false));
+
+    return (() => {
+      navigation.removeListener(onRoomSelected);
+    });
```

Comment:
listeners are not being removed properly see: https://nodejs.org/dist/v11.13.0/docs/api/events.html#events_emitter_removelistener_eventname_listener